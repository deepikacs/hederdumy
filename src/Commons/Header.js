import React, { Component } from 'react';
import { Navbar, Nav, NavItem, MenuItem, NavDropdown, Button,Glyphicon  } from 'react-bootstrap';



class Header extends Component {
  render () {
    return (
      <Navbar>
      {/* <Navbar.Header>
        <Navbar.Brand>
          <a href="#brand"><Glyphicon glyph="glyphicon glyphicon-repeat" /></a>
        </Navbar.Brand>
        <Navbar.Toggle />
      </Navbar.Header> */}
      <Navbar.Collapse>
        <Nav>
          <NavItem eventKey={2} href="#" >
            <i className="fa fa-repeat" aria-hidden="true"></i>
          </NavItem>
        <NavItem eventKey={2} href="#">
        <i className="fa fa-play" aria-hidden="true"></i>
        </NavItem>
        <NavItem eventKey={2} href="#">
        <i className="fa fa-pause" aria-hidden="true">
        </i></NavItem>
        <NavItem eventKey={2} href="#">
        <i className="fa fa-undo" aria-hidden="true">
        </i></NavItem>
          <NavItem eventKey={2} href="#">
          <i className="fa fa-repeat " aria-hidden="true">
          </i></NavItem>
          <NavItem eventKey={2} href="#">
          <i className="fa fa-repeat mycontent-left  " aria-hidden="true"></i>
          </NavItem>
          <NavItem eventKey={2} href="#" className="abc"></NavItem>
          <NavItem eventKey={2} href="#">
          <i className="fa fa-search search" aria-hidden="true">
          </i><input type="text" placeholder="shipment ID/Invoice Number"/>
           </NavItem>
          
          <NavItem eventKey={2} href="">
          <i className="fa fa-chevron-left mycontent-right " aria-hidden="true">
          </i><span className="content">Last 24 hours</span></NavItem>
          <NavItem eventKey={2} href="#">
          <i className="fa fa-chevron-right mycontent-left" aria-hidden="true"></i>June 12th 10:00 hrs to June 13th 10:00 hrs
          </NavItem>
          <NavItem eventKey={2} href="#"><i className="fa fa-file-image-o mycontent-left" aria-hidden="true"></i></NavItem>
        {/* </Nav>
        <Nav pullRight> */}
        <NavItem eventKey={2} href="#">
        <i className="fa fa-share-alt" aria-hidden="true">
        </i></NavItem>
        <NavItem eventKey={2} href="#">
        <i className="fa fa-th-list" aria-hidden="true"></i>
        </NavItem>
        <NavItem eventKey={2} href="#">
        <i className="fa fa-eye-slash" aria-hidden="true"></i>
        </NavItem>
        </Nav>
      </Navbar.Collapse>
    </Navbar>
    )
  }
}

export default Header;