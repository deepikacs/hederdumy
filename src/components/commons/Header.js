import React, { Component } from 'react';
import { Navbar, Nav, NavItem, MenuItem, NavDropdown, Button,Glyphicon,Grid,Row,Col  } from 'react-bootstrap';



class Header extends Component {
  render () {
    return (
      <Navbar expand="md">
        <Grid>
          <Row className="row-edit">
            <Col xs={12} md={12}>
        <Navbar.Collapse>
          <Nav>
            <NavItem eventKey={1} href="#" >
              <i className="fa fa-repeat" aria-hidden="true"></i>
            </NavItem>
            <NavItem eventKey={2} href="#">
              <i className="fa fa-play " aria-hidden="true"></i>
            </NavItem>
            <NavItem eventKey={3} href="#">
              <i className="fa fa-pause" aria-hidden="true"></i>
            </NavItem>
            <NavItem eventKey={4} href="#">
              <i className="fa fa-undo" aria-hidden="true"></i>
            </NavItem>
            <NavItem eventKey={5} href="#">
              <i className="fa fa-undo gly-rotate-252" aria-hidden="true"></i>
            </NavItem>
            <NavItem eventKey={6} href="#">
              <i className="fa fa-undo gly-rotate-80" aria-hidden="true"></i>
            </NavItem>
            <NavItem eventKey={7} href="#" className="verticleline"></NavItem>
            <NavItem eventKey={8} href="#">
              <i className="fa fa-search mginleft-27" aria-hidden="true"></i>
              <input type="text" placeholder="shipment ID/Invoice Number"/>
            </NavItem>
            <NavItem eventKey={9} href="#" className="verticleline"></NavItem>
            <NavItem eventKey={10} href="">
              <i className="fa fa-chevron-left mginleft-27" aria-hidden="true"></i>
              <span className="margin-left20">Last 24 hours</span>
            </NavItem>
            <NavItem eventKey={11} href="#">
              <i className="fa fa-chevron-right marginleft-10" aria-hidden="true"></i>
              <span className="mgnleft15">June 12th 10:00 hrs to June 13th 10:00 hrs</span>
            </NavItem>
            <NavItem eventKey={12} href="#" className="verticleline"></NavItem>
            
            <NavItem eventKey={13} href="#"><i className="fa fa-file-image-o mginleft-27 " aria-hidden="true"></i></NavItem>
          
            <NavItem eventKey={14} href="#" className="verticleline"></NavItem>
            <NavItem eventKey={15} href="#">
              <i className="fa fa-share-alt mginleft-27" aria-hidden="true"></i>
            </NavItem>
            <NavItem eventKey={16} href="#">
              <i className="fa fa-th-list" aria-hidden="true"></i>
            </NavItem>
            <NavItem eventKey={17} href="#">
              <i className="fa fa-database" aria-hidden="true"></i>
            </NavItem>
        </Nav>
      </Navbar.Collapse>
      </Col>
      </Row>
      </Grid>
    </Navbar>

  //   <Navbar collapseOnSelect>
  //   <Navbar.Header>
  //     <Navbar.Brand>
  //       <a href="#brand">React-Bootstrap</a>
  //     </Navbar.Brand>
  //     <Navbar.Toggle />
  //   </Navbar.Header>
  //   <Navbar.Collapse>
  //     <Nav>
  //       <NavItem eventKey={1} href="#">
  //         Link
  //       </NavItem>
  //       <NavItem eventKey={2} href="#">
  //         Link
  //       </NavItem>
  //       <NavDropdown eventKey={3} title="Dropdown" id="basic-nav-dropdown">
  //         <MenuItem eventKey={3.1}>Action</MenuItem>
  //         <MenuItem eventKey={3.2}>Another action</MenuItem>
  //         <MenuItem eventKey={3.3}>Something else here</MenuItem>
  //         <MenuItem divider />
  //         <MenuItem eventKey={3.3}>Separated link</MenuItem>
  //       </NavDropdown>
  //       <NavItem eventKey={1} href="#" >
  //             <i className="fa fa-repeat" aria-hidden="true"></i>
  //           </NavItem>
  //           <NavItem eventKey={2} href="#">
  //             <i className="fa fa-play " aria-hidden="true"></i>
  //           </NavItem>
  //           <NavItem eventKey={3} href="#">
  //             <i className="fa fa-pause" aria-hidden="true"></i>
  //           </NavItem>
  //           <NavItem eventKey={4} href="#">
  //             <i className="fa fa-undo" aria-hidden="true"></i>
  //           </NavItem>
  //           <NavItem eventKey={5} href="#">
  //             <i className="fa fa-undo gly-rotate-252" aria-hidden="true"></i>
  //           </NavItem>
  //           <NavItem eventKey={6} href="#">
  //             <i className="fa fa-undo gly-rotate-80" aria-hidden="true"></i>
  //           </NavItem>
  //           <NavItem eventKey={7} href="#" className="verticleline"></NavItem>
  //           <NavItem eventKey={8} href="#">
  //             <i className="fa fa-search mginleft-27" aria-hidden="true"></i>
  //             <input type="text" placeholder="shipment ID/Invoice Number"/>
  //           </NavItem>
  //           <NavItem eventKey={9} href="#" className="verticleline"></NavItem>
  //           <NavItem eventKey={10} href="">
  //             <i className="fa fa-chevron-left mginleft-27" aria-hidden="true"></i>
  //             <span className="margin-left20">Last 24 hours</span>
  //           </NavItem>
  //           <NavItem eventKey={11} href="#">
  //             <i className="fa fa-chevron-right marginleft-10" aria-hidden="true"></i>
  //             <span className="mgnleft15">June 12th 10:00 hrs to June 13th 10:00 hrs</span>
  //           </NavItem>
  //           <NavItem eventKey={12} href="#" className="verticleline"></NavItem>
            
  //           <NavItem eventKey={13} href="#"><i className="fa fa-file-image-o mginleft-27 " aria-hidden="true"></i></NavItem>
          
  //           <NavItem eventKey={14} href="#" className="verticleline"></NavItem>
  //           <NavItem eventKey={15} href="#">
  //             <i className="fa fa-share-alt mginleft-27" aria-hidden="true"></i>
  //           </NavItem>
  //           <NavItem eventKey={16} href="#">
  //             <i className="fa fa-th-list" aria-hidden="true"></i>
  //           </NavItem>
  //           <NavItem eventKey={17} href="#">
  //             <i className="fa fa-database" aria-hidden="true"></i>
  //           </NavItem>
  //     </Nav>
  //     <Nav pullRight>
  //       <NavItem eventKey={1} href="#">
  //         Link Right
  //       </NavItem>
  //       <NavItem eventKey={2} href="#">
  //         Link Right
  //       </NavItem>
  //     </Nav>
  //   </Navbar.Collapse>
  // </Navbar>
    )
  }
}

export default Header;