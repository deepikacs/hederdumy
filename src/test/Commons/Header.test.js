import React from 'react';
import ReactDOM from 'react-dom';
import { shallow, mount, render, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Header from '../../components/commons/Header';

// global.localStorage = new LocalStorageMock;

configure({ adapter: new Adapter() });

describe('Header', () => {
it('should mount Header component', () => {
    const wrapper = shallow(<Header />)
    expect(toJson(wrapper)).toMatchSnapshot();
  })
})