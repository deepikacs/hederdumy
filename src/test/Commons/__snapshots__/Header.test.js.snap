// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should mount Header component 1`] = `
<Uncontrolled(Navbar)
  bsStyle="default"
  expand="md"
>
  <Grid
    bsClass="container"
    componentClass="div"
    fluid={false}
  >
    <Row
      bsClass="row"
      className="row-edit"
      componentClass="div"
    >
      <Col
        bsClass="col"
        componentClass="div"
        md={12}
        xs={12}
      >
        <NavbarCollapse>
          <Nav
            bsClass="nav"
            justified={false}
            pullLeft={false}
            pullRight={false}
            stacked={false}
          >
            <NavItem
              active={false}
              disabled={false}
              eventKey={1}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-repeat"
              />
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={2}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-play "
              />
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={3}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-pause"
              />
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={4}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-undo"
              />
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={5}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-undo gly-rotate-252"
              />
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={6}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-undo gly-rotate-80"
              />
            </NavItem>
            <NavItem
              active={false}
              className="verticleline"
              disabled={false}
              eventKey={7}
              href="#"
            />
            <NavItem
              active={false}
              disabled={false}
              eventKey={8}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-search mginleft-27"
              />
              <input
                placeholder="shipment ID/Invoice Number"
                type="text"
              />
            </NavItem>
            <NavItem
              active={false}
              className="verticleline"
              disabled={false}
              eventKey={9}
              href="#"
            />
            <NavItem
              active={false}
              disabled={false}
              eventKey={10}
              href=""
            >
              <i
                aria-hidden="true"
                className="fa fa-chevron-left mginleft-27"
              />
              <span
                className="margin-left20"
              >
                Last 24 hours
              </span>
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={11}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-chevron-right marginleft-10"
              />
              <span
                className="mgnleft15"
              >
                June 12th 10:00 hrs to June 13th 10:00 hrs
              </span>
            </NavItem>
            <NavItem
              active={false}
              className="verticleline"
              disabled={false}
              eventKey={12}
              href="#"
            />
            <NavItem
              active={false}
              disabled={false}
              eventKey={13}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-file-image-o mginleft-27 "
              />
            </NavItem>
            <NavItem
              active={false}
              className="verticleline"
              disabled={false}
              eventKey={14}
              href="#"
            />
            <NavItem
              active={false}
              disabled={false}
              eventKey={15}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-share-alt mginleft-27"
              />
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={16}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-th-list"
              />
            </NavItem>
            <NavItem
              active={false}
              disabled={false}
              eventKey={17}
              href="#"
            >
              <i
                aria-hidden="true"
                className="fa fa-database"
              />
            </NavItem>
          </Nav>
        </NavbarCollapse>
      </Col>
    </Row>
  </Grid>
</Uncontrolled(Navbar)>
`;
